packages/components/src/toolbar/toolbar/types.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import type { ToolbarProps as ReakitToolbarContainerProps } from 'reakit/Toolbar';
+
+export type ToolbarProps = {
+	/**
+	 * Class to set on the container div.
+	 */
+	className?: string;
+	/**
+	 * An accessible label for the toolbar.
+	 */
+	label: string;
+} & Partial< ReakitToolbarContainerProps >;
```

Comment:
I don't quite think we should be adding `ReakitToolbarContainerProps` here, because a lot of them aren't intended for the consumer of the wp-components Toolbar to provide. They are intended for Reakit's `useToolbarState` hook to provide. There may be _some_ props that are legitimately useful, but there's also a benefit to controlling our own API surface in an intentional way so we don't lock ourselves into the Reakit implementation.

What do you think about keeping `ToolbarProps` really simple, like maybe `label` and `children` only, and just add in generic `div` props to be passed through? And we can always add more official props to the documentation when necessary.

```tsx
// toolbar/index.tsx
function UnforwardedToolbar(
	{
		className,
		label,
		...props
	}: WordPressComponentProps< ToolbarProps, 'div', false >, // 👈

// toolbar-container.tsx	
function UnforwardedToolbarContainer(
	{ label, ...props }: WordPressComponentProps< ToolbarProps, 'div', false >, // 👈
	ref: ForwardedRef< any >
```

Reply:
This is much cleaner, @mirka! Great suggestion, covered in [2cf63de](https://github.com/WordPress/gutenberg/pull/47087/commits/2cf63de78e044147e5fc5fb74842521aac224b19).